config.go

```diff
@@ -321,6 +326,12 @@ func loadConfig() (*config, error) {
 		cfg.DcrdServ = defaultHost + ":" + activeNet.JSONRPCClientPort
 	}
 
+	//Set the host name and port of the walletserver
+	//if the user does not specify them
+	if cfg.WalletSever == "" {
+		cfg.WalletSever = defaultWalletServer
```

Comment:
The default port would actually depend on if `--testnet` or `--simnet` were used.  I suggest changing the default servers to not include the port, and then append the port for the chosen net.